dev-lang/jimtcl/files/jimtcl-0.78-bootstrap.patch

```diff
@@ -0,0 +1,16 @@
+diff --git a/autosetup/autosetup-find-tclsh b/autosetup/autosetup-find-tclsh
```

Comment:
Having some kind of justification in commit message would be nice.

Reply:
This ensures stable code path in autosetup scripts.
Commit message is updated.